api/controllers/comment.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const winston = require('winston');
+const request = require('request');
+const axios = require('axios');
+
+const config = require('../config');
+const logger = winston.createLogger(config.logger.winston);
+const db = require('../models');
+const common = require('../common');
+
+
+/**
+ * @apiGroup Comment
+ * @api {get} /comment  Get Comment List of Particular Project
+ * @apiHeader {String} authorization 
+ *                              A valid JWT token "Bearer: xxxxx"
+ * @apiSuccess {Object}            List of Comments 
+ */
+
+router.get('/', common.jwt({credentialsRequired: true}), (req, res, next) =>{
+    if(!req.query.project) return next("project not defined");
+    let project = req.query.project;
+    if(project) {
```

Comment:
You are forgetting to do something very important here... No access control!! :(

You should 1) find the project 2) make sure the project exists and it's not removed, etc.. 3) make sure the user has access to the project (they should have read access to the project)

See other similar API as a reference.